server/src/routes/auth.js

```diff
@@ -0,0 +1,14 @@
+import express from 'express';
+import { verifyGoogleToken, verifyFBToken } from '../controllers/auth';
+
+const router = express.Router();
+
+router.post('/login', async (req, res) => {
+  const result = req.body.method === 'google'
+    ? await verifyGoogleToken(req.body.access_token)
```

Comment:
Handle google login for now.
We can easyly add new ones later.

Reply:
also, using ternary here is not a good idea.
This will break when re.body.method = "Chirag"